kustomize/overlay/7.1.0/securebanking/defaults/delete-ingress.yaml

```diff
@@ -0,0 +1,17 @@
+$patch: delete
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: end-user-ui
+---
+$patch: delete
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: login-ui
+---
+$patch: delete
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: platform-ui
```

Comment:
Why do we need to delete these?

Reply:
We aren't using them in our deployments, we can keep them and see if they are of any use?